src/Advection/weno_tables.jl

```diff
@@ -0,0 +1,217 @@
+@inline coeff_β(scheme::WENO{4, FT}, ::Val{2}) where FT = FT.((547,  -2522, 1922, -494,  3443,  -5966,  1602, 2843,  -1642,  267))
+@inline coeff_β(scheme::WENO{4, FT}, ::Val{3}) where FT = FT.((2107, -9402, 7042, -1854, 11003, -17246, 4642, 7043,  -3882,  547))
+
+@inline coeff_β(scheme::WENO{5, FT}, ::Val{0}) where FT = FT.((22658,  -208501, 364863, -288007, 86329, 482963,  -1704396, 1358458, -411487, 1521393, -2462076, 758823, 1020563, -649501, 107918))
+@inline coeff_β(scheme::WENO{5, FT}, ::Val{1}) where FT = FT.((6908,   -60871,  99213,  -70237,  18079, 138563,  -464976,  337018,  -88297,  406293,  -611976,  165153, 242723,  -140251,  22658))
+@inline coeff_β(scheme::WENO{5, FT}, ::Val{2}) where FT = FT.((6908,   -51001,  67923,  -38947,  8209,  104963,  -299076,  179098,  -38947,  231153,  -299076,  67923,  104963,  -51001,    6908))
+@inline coeff_β(scheme::WENO{5, FT}, ::Val{3}) where FT = FT.((22658,  -140251, 165153, -88297,  18079, 242723,  -611976,  337018,  -70237,  406293,  -464976,  99213,  138563,  -60871,    6908))
+@inline coeff_β(scheme::WENO{5, FT}, ::Val{4}) where FT = FT.((107918, -649501, 758823, -411487, 86329, 1020563, -2462076, 1358458, -288007, 1521393, -1704396, 364863, 482963,  -208501,  22658))
+
+@inline coeff_β(scheme::WENO{6, FT}, ::Val{0}) where FT = FT.((1152561, -12950184, 29442256, -33918804, 19834350, -4712740, 36480687, -166461044, 192596472, -113206788, 27060170, 190757572, -444003904, 262901672, -63394124, 260445372, -311771244, 76206736, 94851237, -47460464, 6150211))
+@inline coeff_β(scheme::WENO{6, FT}, ::Val{1}) where FT = FT.((271779,  -3015728,  6694608,  -7408908,  4067018,  -880548,  8449957,  -37913324,  42405032,  -23510468,  5134574,  43093692,  -97838784,  55053752,  -12183636, 56662212,  -65224244,  14742480, 19365967, -9117992,  1152561))
+@inline coeff_β(scheme::WENO{6, FT}, ::Val{2}) where FT = FT.((139633,  -1429976,  2863984,  -2792660,  1325006,  -245620,  3824847,  -15880404,  15929912,  -7727988,   1458762,  17195652,  -35817664,  17905032,  -3462252,  19510972,  -20427884,  4086352,  5653317,  -2380800,  271779))
+@inline coeff_β(scheme::WENO{6, FT}, ::Val{3}) where FT = FT.((271779,  -2380800,  4086352,  -3462252,  1458762,  -245620,  5653317,  -20427884,  17905032,  -7727988,   1325006,  19510972,  -35817664,  15929912,  -2792660,  17195652,  -15880404,  2863984,  3824847,  -1429976,  139633))
+@inline coeff_β(scheme::WENO{6, FT}, ::Val{4}) where FT = FT.((1152561, -9117992,  14742480, -12183636, 5134574,  -880548,  19365967, -65224244,  55053752,  -23510468,  4067018,  56662212,  -97838784,  42405032,  -7408908,  43093692,  -37913324,  6694608,  8449957,  -3015728,  271779))
+@inline coeff_β(scheme::WENO{6, FT}, ::Val{5}) where FT = FT.((6150211, -47460464, 76206736, -63394124, 27060170, -4712740, 94851237, -311771244, 262901672, -113206788, 19834350, 260445372, -444003904, 19259647,  -33918804, 190757572, -166461044, 29442256, 36480687, -12950184, 1152561))
```

Comment:
How are these being calculated? Can they be generated with metaprogramming?

Reply:
they are generated from 
$
\beta_r = \sum_{l=1}^{k-1} \int_{x_{i-1/2}}^{x_{i+1/2}} \Delta x^{2l-1} \left( \frac{\partial^l p_r(x)}{\partial x^l} \right)^2 dx
$

Reply:
p_r are polynomials?

Reply:
yep, they are the reconstruction polynomials of stencil `r`

Reply:
Are these coefficients only valid for staggered reconstruction, or will they also work for co-located reconstruction (which was proposed in the now-closed #2388)